### PR TITLE
Counting total number of likes on a student's reviews

### PR DESCRIPTION
**Pull request by juliejeong**
### Summary 
I created a new endpoint _getTotalLikesByStudentId_ that takes a student's net id and returns the total number of likes the student has gotten on their reviews.

### Test Plan 

I created a test file with testing data of students and reviews and checked that the endpoint correctly returns the total number of likes.

### CODE REVIEW

File: server/endpoints/Profile.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      const reviews: ReviewDocument[] = await Promise.all(
+        (reviewIds).map(async (reviewId) => await Reviews.findOne({ _id: reviewId })),
+      );
+      reviews.forEach((review) => {
+        if ('likes' in review) {
+          totalLikes += review.likes;
+        }
+      });
+
+      return { resCode: 1, totalLikes };
```
Comment (AkashAryal) [inline server/endpoints/Profile.ts:27]: Lets start standardizing the return object of our end points. In Michelle's PR that was merged in earlier (https://github.com/cornell-dti/course-reviews-react-2.0/pull/280/files) we went with `{code: <> , message:<>}` where message contains data if there is no error. Lets try that here too

File: server/endpoints/Profile.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.log("Error: at 'getTotalLikesByStudentId' method");
+      // eslint-disable-next-line no-console
+      console.log(error);
+      return { resCode: 0 };
```
Comment (AkashAryal) [inline server/endpoints/Profile.ts:33]: Same here

**Review by AkashAryal — CHANGES_REQUESTED** [review]
Looks good overall Julie! Just a few small consistency changes. Good job!

File: server/endpoints/Profile.ts (hunk 1 above)
Comment (juliejeong) [reply server/endpoints/Profile.ts]: fixed!

File: server/endpoints/Profile.test.ts [hunk 3]
```diff
@@ -68,8 +118,27 @@ const testUsers: Student[] = [
     reviews: null,
     likedReviews: [],
   },
+  /*
```
Comment (AkashAryal) [inline server/endpoints/Profile.test.ts:121]: Let's delete if we don't need this

**Review by AkashAryal — CHANGES_REQUESTED** [review]
Looks good! Just delete the commented out code and it should be good. The tests pass locally but we seem to get a mongo index error here in ci. I will probably just override and force merge it.

File: server/endpoints/Profile.test.ts (hunk 3 above)
Comment (juliejeong) [reply server/endpoints/Profile.test.ts]: Fixed!

**Pull request merged**